app/src/main/java/com/twilio/video/app/ui/room/VideoService.kt

```diff
@@ -3,39 +3,23 @@ package com.twilio.video.app.ui.room
 import android.app.Service
 import android.content.Context
 import android.content.Intent
+import android.os.Binder
```

Comment:
I saw below description in the issue ticket. Do you think we need to do this?
```
This can be fixed using android:stopWithTask="true" in the service declaration in the app manifest
```

Reply:
No, that suggestions is not needed. Actually, that's the default value for the service declaration in the `AndroidManifest.xml`